superset/config.py

```diff
@@ -841,7 +841,7 @@ class D3Format(TypedDict, total=False):
 # Console Log Settings
 
 LOG_FORMAT = "%(asctime)s:%(levelname)s:%(name)s:%(message)s"
-LOG_LEVEL = "DEBUG"
+LOG_LEVEL = "INFO"
```

Comment:
I think that's already done in `master`, also realized back then that `logging.INFO != "INFO"` ....